database/src/seeds/01_db_system_users.ts

```diff
@@ -28,12 +27,12 @@ const systemUsers = [
   { identifier: 'rstens', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.SYSTEM_ADMINISTRATOR },
   { identifier: 'zochampi', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.SYSTEM_ADMINISTRATOR },
   { identifier: 'test1', type: SYSTEM_IDENTITY_SOURCE.BCEID, roleId: SYSTEM_USER_ROLE_ID.SYSTEM_ADMINISTRATOR },
-  { identifier: 'test2', type: SYSTEM_IDENTITY_SOURCE.BCEID, roleId: SYSTEM_USER_ROLE_ID.CREATOR },
+  { identifier: 'test2', type: SYSTEM_IDENTITY_SOURCE.BCEID, roleId: 0 },
   { identifier: 'test3', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.SYSTEM_ADMINISTRATOR },
   { identifier: 'test4', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.SYSTEM_ADMINISTRATOR },
   { identifier: 'test5', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.DATA_ADMINISTRATOR },
-  { identifier: 'test6', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.CREATOR },
-  { identifier: 'test7', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: SYSTEM_USER_ROLE_ID.CREATOR },
+  { identifier: 'test6', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: 0 },
+  { identifier: 'test7', type: SYSTEM_IDENTITY_SOURCE.IDIR, roleId: 0 },
```

Comment:
Lets just remove the roleId field altogether or just set it to null or something.  Just so zero isn't confused with an actual role id.

Reply:
And then on line 65 below, lets check if the roleid exists, and if not, skip adding any system role record for that user.

Reply:
super nit: Maybe tweak `// Add system administrator role` to just `// Add system role`